sdks/python/apache_beam/portability/__init__.py

```diff
@@ -17,3 +17,18 @@
 
 """For internal use only; no backwards-compatibility guarantees."""
 from __future__ import absolute_import
+
+from typing import TYPE_CHECKING
+from typing import NamedTuple
+
+if TYPE_CHECKING:
+  from apache_beam.portability.api import metrics_pb2
+
+
+PropertiesFromEnumValue = NamedTuple(
```

Comment:
yeah, I typically follow that convention as well.  In this case, common_urns.py would cause an import cycle, because it imports the _urn.py files which need access to `PropertiesFromEnumValue`.  

I considered putting `PropertiesFromEnumValue` in a new module, but it felt like overkill.  Happy to do so, if it's preferable.

Reply:
@udim @robertwb Let me know if you prefer to move `PropertiesFromEnumValue` this into a separate module, and if so what you think a good name would be.

Reply:
I'll defer to Udi on this, I have no strong opinion either way.

Reply:
Yes, I would prefer putting this in a module. How about the existing `apache_beam.utils.proto_utils` or `apache_beam.utils.urns`?

Reply:
I believe it has to be below `apache_beam.portability` because when the urn generation runs we are in the middle of the beam build process, and IIRC it's not reliable to import `apache_beam` at this point.  So the solve was to treat the `portability.api` package as the root package to import and inspect only the generated pb2 modules.   So it would need to be something like `apache_beam.portability.utils`.

Reply:
I went with `apache_beam.portability.utils`.  Let me know  if you have any concerns with that.